browsers/samsunginternet_android.json

```diff
@@ -69,10 +69,14 @@
         },
         "7.2": {
           "release_date": "2018-03-07",
```

Comment:
This also looks wrong as [the official announcement](https://medium.com/samsung-internet-dev/7-2-stable-is-here-81fdbfca75b4) was in June 20.